frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx

```diff
@@ -68,12 +68,14 @@ import { SectionFieldRow } from './SectionFieldRow'
 export interface FieldRowContainerProps {
   field: FormFieldDto
   index: number
+  isVisible: boolean
   isDraggingOver: boolean
 }
 
 export const FieldRowContainer = ({
   field,
   index,
+  isVisible,
```

Comment:
I'm thinking this property at a JSX.Element is a bit confusing, since `visibility` means something in CSS 🤔 .

Can we find a clearer name to represent what this is? Maybe something like `isHiddenByLogicRule` or similar?

Reply:
Alright! Will change to `isHiddenByLogic`.